onomy/chain.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "onomy",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://onomy.io/",
+  "pretty_name": "Onomy",
+  "chain_id": "onomy-mainnet-1",
+  "bech32_prefix": "onomy",
+  "daemon_name": "onomyd",
+  "node_home": "$HOME/.onomyd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
```

Comment:
do we have fee range token data? e.g.,
  "fixed_min_gas_price": 0,
  "low_gas_price": 0,
  "average_gas_price": 0.025,
  "high_gas_price": 0.04
Osmosis will need this anyway.

Reply:
Updated